sympy/simplify/cse_main.py

```diff
@@ -692,6 +692,8 @@ def cse(exprs, symbols=None, optimizations=None, postprocess=None,
     if isinstance(exprs, (Basic, MatrixBase)):
         exprs = [exprs]
 
+    exprs = sympify(exprs)
```

Comment:
This should probably be before the above check so that things like `cse(1)` work too.

Reply:
Thanks. Fixed.